bambi/interpret/helpers.py

```diff
@@ -0,0 +1,215 @@
+from typing import Union
+import warnings
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from arviz import InferenceData
+
+from bambi import Model
+from bambi.interpret.create_data import create_grid
+from bambi.interpret.utils import ConditionalInfo, VariableInfo
+
+# ignore ArviZ user warning about 'data' not being in the InferenceData group scheme
+warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
Do we know what's the reach of this filter? If it works at the session level, we may be ignoring warnings on behalf of the users and that's not ideal.

Reply:
That's a good point. This would ignore all UserWarnings. However, we can ignore specific warnings via

```python3
warnings.filterwarnings("ignore", message="The group data is not defined in the InferenceData scheme")
```

which is what I will change it to.

Reply:
I like that